static/app/components/charts/miniBarChart.tsx

```diff
@@ -173,6 +174,14 @@ function MiniBarChart({
         showMinLabel: true,
         showMaxLabel: true,
         interval: Infinity,
+        axisLabel: {
+          formatter(value: number) {
+            if (tooltipFormatter) {
+              return tooltipFormatter(value);
+            }
+            return formatAbbreviatedNumber(value);
```

Comment:
```suggestion
            tooltipFormatter?.(value) ?? formatAbbreviatedNumber(value)
```

just a suggestion but i'm totally indifferent, maybe too clever

Reply:
I dunno, I think that might be getting clever and it will likely transpile to something larger than the if would.

Reply:
Yeah true